cluster/userdata-master.yaml

```diff
@@ -123,8 +121,9 @@ coreos:
         --rkt-path=/usr/bin/rkt \
         --register-schedulable=false \
         --allow-privileged \
-        --node-labels=master=true \
+        --node-labels=node-role.kubernetes.io/master,node.alpha.kubernetes.io/role=master \
```

Comment:
These labels are taken from `kube-aws`.

With this you can better see the type in the node list:

```console
$ kubectl get nodes -L node.alpha.kubernetes.io/role
NAME                                             STATUS    AGE       VERSION           ROLE
ip-172-31-0-127.eu-central-1.compute.internal    Ready     9m        v1.7.5+coreos.0   worker
ip-172-31-1-130.eu-central-1.compute.internal    Ready     1h        v1.7.5+coreos.0   master
```

But still be able to filter without using a key/value pair

```console
$ kubectl get nodes -l node-role.kubernetes.io/master
NAME                                             STATUS    AGE       VERSION
ip-172-31-1-130.eu-central-1.compute.internal    Ready     1h        v1.7.5+coreos.0
ip-172-31-16-219.eu-central-1.compute.internal   Ready     1h        v1.7.5+coreos.0
```

We could think about dropping one of the labels.